charts/prometheus/templates/prometheus-service.yaml

```diff
@@ -41,3 +35,29 @@ spec:
       port: {{ $.Values.ports.http }}
       targetPort: {{ $.Values.ports.http }}
 {{ end }}
+################################
+## Prometheus Service
+## load balancing between all pods
+#################################
```

Comment:
@sjmiller609 nit, but should this comment go under the `---`, since its part of the next document?